fern/api-description-overwrites.yml

```
@@ -229,207 +199,191 @@ paths:
+      summary: Delete a snapshot (shard)
       tags:
       - Snapshots
     get:
-      description: Download specified snapshot of a shard from a collection as a file
-      summary: Download collection snapshot
+      description: Downloads the specified snapshot of a shard from a collection as a file.
+      summary: Download a snapshot (shard)
       tags:
       - Snapshots
   /collections/{collection_name}/snapshots:
     get:
-      description: Get list of snapshots for a collection
-      summary: List collection snapshots
+      description: Retrieves a list of all snapshots for a specified collection.
+      summary: List all snapshots (collection)
       tags:
       - Snapshots
     post:
-      description: Create new snapshot for a collection
-      summary: Create collection snapshot
+      description: Creates a new snapshot for a specified collection.
+      summary: Create a snapshot (collection)
       tags:
       - Snapshots
   /collections/{collection_name}/snapshots/recover:
     put:
-      description: Recover local collection data from a snapshot. This will overwrite
-        any data, stored on this node, for the collection. If collection does not
-        exist - it will be created.
-      summary: Recover from a snapshot
+      description: Recovers local collection data from a snapshot. This will overwrite any collection data stored on the node. If the collection does not exist, it will be created.
+      summary: Recover from a snapshot (collection)
       tags:
       - Snapshots
   /collections/{collection_name}/snapshots/upload:
     post:
-      description: Recover local collection data from an uploaded snapshot. This will
-        overwrite any data, stored on this node, for the collection. If collection
-        does not exist - it will be created.
-      summary: Recover from an uploaded snapshot
+      description: Recovers local collection data from an uploaded snapshot. This will overwrite any collection data stored on the node. If the collection does not exist, it will be created.
+      summary: Recover from an uploaded snapshot (collection)
       tags:
       - Snapshots
   /collections/{collection_name}/snapshots/{snapshot_name}:
     delete:
-      description: Delete snapshot for a collection
-      summary: Delete collection snapshot
+      description: Deletes the specified snapshot for a collection.
+      summary: Delete a snapshot (collection)
       tags:
       - Snapshots
     get:
-      description: Download specified snapshot from a collection as a file
-      summary: Download collection snapshot
+      description: Downloads the specified snapshot file from a collection.
+      summary: Download a snapshot (collection)
       tags:
       - Snapshots
   /healthz:
     get:
-      description: Checks the health of individual service instance.
-      summary: Kubernetes healthz endpoint
+      description: Monitors the health of individual containers.
```

```suggestion
      description: Checks the health of individual instance.
```

Monitoring is more of a continuous process, this one is more of a single check. Also it is not necessary a container